map/framework.cpp

```diff
@@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
 
 void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
```

Comment:
Почему не должен? Ведь он дергает специальный внешний колбэк во фреймворке. Почему документация к этрму колбэку не может гарантировать его запуск только на главном потоке? И даже тред чекер можно вставить.

> On 25.3.2016, at 23.03, ygorshenin <notifications@github.com> wrote:
> 
> In map/framework.cpp:
> 
> > @@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
> >  
> >  void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
> Вот это Сергей и сделал здесь - там где вызывается Deregister зовется RunOnGUIThread. В MwmSet это пихать нельзя и неправильно, так как он ни про какие GUI Threads не знает и не должен.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>